src/helpers/validate-expected-inject-params.js

```diff
@@ -0,0 +1,39 @@
+/***************************************************************************************
+ * (c) 2025 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ ****************************************************************************************/
+
+// reads the function signature and sees if a caller doesn't pass enough parameters
+module.exports = function autoValidate(fn) {
```

Comment:
for each module that used to use `inject-loader`, their default exports are now wrapped in this check that ensures they've been passed the correct injectable params.